map-demo/app/views/map.scala.html

```diff
@@ -12,12 +12,22 @@
                 <strong class="phase-tag">BETA</strong>
                 <span>This is a new service – your <a href="#">feedback</a> will help us to improve it.</span>
             </p>
+            <div id="location-input">
+                <label class="form-label" for="location-search" id="location-text-field">
+                    Search for Location
+                </label>
+                <input class="form-control" id="location-search" type="text">
+                <a class="button button-start" id="search-button" href="#" role="button" onclick="geocode(document.getElementById('location-search').value)">Search</a>
+            </div>
```

Comment:
Can this be moved out of the phase-banner div? Maybe in to it's own seperate div section.
The search button is a little large, drop the 'button-start' class.

Some spacing between header and map would be nice!